commerce-theme-minium/commerce-theme-minium/static-version/src/styles/components/add-to-cart-button.scss

```diff
@@ -198,21 +198,45 @@
 			font-size: 12px;
 			height: 60px;
 			min-width: 160px;
+			position: relative;
 
-			&.commerce-button--good {
-				&:hover {
-					.added-quantity {
-						display: none;
-						height: inherit;
-						width: inherit;
-					}
-				}
+			.added-quantity {
+				display: none;
+			}
 
+			&.commerce-button--good {
 				&:hover::before {
 					align-items: center;
+					animation: addMoreHover .1s ease-out 1 forwards;
+					background-color: $color-theme;
+					border-radius: 8px;
 					content: attr(data-add-more-label);
```

Comment:
@gianmarcobrunialti what is doing this line : content: attr(data-add-more-label); ?

Reply:
It's grabbing the data-attribute (`data-add-more-label`) from the element (rendered through the soy template, by which we are using i18n properties) to populate the content of its `::before` pseudo-element.